pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1385,6 +1429,44 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     }
   }
 
+  async readFile(filePath: string, options?: { encoding?: BufferEncoding }): Promise<string> {
+    const encoding = options?.encoding ?? 'utf-8';
+    const stdout: Buffer[] = [];
+    const stderr: Buffer[] = [];
+
+    try {
+      // Use limaSpawn to avoid logging file contents (too verbose).
+      const proc = this.limaSpawn({ root: true }, ['/bin/cat', filePath]);
+
+      await new Promise<void>((resolve, reject) => {
+        proc.stdout?.on('data', (chunk: Buffer | string) => {
+          stdout.push(Buffer.from(chunk));
+        });
+        proc.stderr?.on('data', (chunk: Buffer | string) => {
+          stderr.push(Buffer.from(chunk));
+        });
+        proc.on('error', reject);
+        proc.on('exit', (code, signal) => {
+          if (code || signal) {
+            return reject(`Failed to read ${ filePath }: cat exited with ${ code || signal }`);
```

Comment:
Should be `/bin/cat on the VM` just to make life easier for anyone dealing with this error message.

Reply:
Changed in f363bc2f175535a251c9912abc3084b467b36439